great_tables/_text.py

```diff
@@ -26,24 +26,53 @@ def _md_html(x: str) -> str:
     return re.sub(r"^<p>|</p>\n$", "", str)
 
 
-def _process_text(x: str | Text | None) -> str:
+def _md_latex(x: str) -> str:
+
+    # TODO: Implement commonmark to LaTeX conversion (through a different library as
+    # commonmark-py does not support it)
+    raise NotImplementedError("Markdown to LaTeX conversion is not supported yet")
+
+
+def _process_text(x: str | Text | None, context: str = "html") -> str:
+
     from great_tables._helpers import UnitStr
 
     if x is None:
         return ""
 
-    if isinstance(x, Md):
-        return _md_html(x.text)
-    elif isinstance(x, Html):
-        return x.text
-    elif isinstance(x, str):
-        return _html_escape(x)
-    elif isinstance(x, Text):
-        return x.text
-    elif isinstance(x, UnitStr):
-        return x.to_html()
+    if context == "html":
+
+        if isinstance(x, Md):
+            return _md_html(x.text)
+        elif isinstance(x, Html):
+            return x.text
+        elif isinstance(x, str):
+            return _html_escape(x)
+        elif isinstance(x, Text):
+            return x.text
+        elif isinstance(x, UnitStr):
+            return x.to_html()
```

Comment:
What if we made all text type x context rendering functions methods?

For example:

* text type - Md, context - html: `Md.to_html()`
* text type - Md, context - latex: `Md.to_latex()`
* text type - Text, context - latex: `Text.to_latex()`
  - This could just return the raw text

This way, no matter whether you get Md, Text, or HTML, you can just call `.to_html()` or `.to_latex()`, so we don't need to do specific isinstance checks on them.  (the `.to_<context>()` becomes an interface).